android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TranslationsBrowserState.kt

```diff
@@ -4,16 +4,19 @@
 
 package mozilla.components.browser.state.state
 
+import mozilla.components.concept.engine.translate.LanguageSetting
 import mozilla.components.concept.engine.translate.TranslationError
 
 /**
  * Value type that represents the state of the translations engine within a [BrowserState].
  *
  * @property isEngineSupported Whether the translations engine supports the device architecture.
+ * @property languageSettings The automatic language settings of the translations engine.
```

Comment:
Might be worthwhile to describe the `Map` here similar to https://github.com/mozilla-mobile/firefox-android/pull/5726/files#diff-4dd92b761139e3e0ebeb8bcacaa1162d5708fc809f6b65dd2f88bcf9b5d208f1R1040 since it won't be immediately clear what the `String` key here represents.

Reply:
Thanks, good catch!